syntax = "proto3";
service TaskService {
  rpc createTask(TaskRequest) returns (TaskResponse);
  rpc getTasks(ListRequest) returns (TaskListResponse);
  rpc modifyTask(TaskRequest) returns (TaskResponse);
}
  message TaskRequest {
    int32 id = 1;
    string title = 2;
  }
  message ListRequest {
    string taskId = 1;
  }
  enum Status {
    RESOLVED = 0;
    UNRESOLVED = 1;
  }
  message Task {
    int32 id = 1;
    string title = 2;
    string color = 3;
    Status resolved = 4;
  }
  message TaskResponse {
    Task Task = 1;
  }
  message TaskListResponse {
    repeated Task Task = 1;
  }
  